test/Integration/IntegrationTestBase.t.sol

```diff
@@ -52,7 +54,14 @@ contract IntegrationTest is Test {
     bytes32 constant REVERSE_LABEL = 0xdec08c9dbbdd0890e300eb5062089b2d4b1c40e3673bbccb5423f7b37dcf9a9c;
     bytes32 constant ADDR_LABEL = 0xe5e14487b78f85faa6e1808e89246cf57dd34831548ff2e6097380d98db2504a;
 
-    function setUp() public {
+    uint256 constant LAUNCH_AUCTION_START_PRICE = 100 ether;
+    uint256 constant LAUNCH_AUCTION_DURATION_HOURS = 36;
+    uint256 constant LAUNCH_TIME = 1723420800; // Monday, August 12, 2024 12:00:00 AM GMT
+
+    uint256 constant EXPIRY_AUCTION_START_PRICE = 1000 ether;
+    uint256 constant EXPIRY_AUCTION_DURATION_DAYS = 21;
```

Comment:
@stevieraykatz do the deploy scripts need to be updated? `DeployPriceOracle` still uses 28 days there, 21 days here.

I'm currently having these assumptions, can you confirm if they're true for the actual deployment?

1. the base prices will be the same for both oracles.
2. the prod oracle will replace the launch oracle without having a sudden change in the `rentPrice`. Practically, this means when both premiums match, I assume you do this when both premiums are 0 after the max of both auction duration times (21 days). Also need to do this before you can re-register an account, before `MIN_REGISTRATION_DURATION + GRACE_PERIOD`

Reply:
Yeah that script is outdated. We use internal tooling to deploy. Exp. premium price oracle will use 21-day duration.

Reply:
ok, makes sense to me then